server/log_rpc_server_test.go

```diff
@@ -350,95 +348,90 @@ func TestQueueLeavesBeginFailsCausesError(t *testing.T) {
 	test.executeBeginFailsTest(t, queueRequest0.LogId)
 }
 
-func TestGetLatestSignedLogRootBeginFails(t *testing.T) {
-	ctrl := gomock.NewController(t)
-	defer ctrl.Finish()
-
-	mockStorage := storage.NewMockLogStorage(ctrl)
-	mockStorage.EXPECT().SnapshotForTree(gomock.Any(), getLogRootRequest1.LogId).Return(nil, errors.New("TX"))
-
-	registry := extension.Registry{
-		AdminStorage: mockAdminStorage(ctrl, getLogRootRequest1.LogId),
-		LogStorage:   mockStorage,
-	}
-	server := NewTrillianLogRPCServer(registry, fakeTimeSource)
-
-	_, err := server.GetLatestSignedLogRoot(context.Background(), &getLogRootRequest1)
-	if err == nil || !strings.Contains(err.Error(), "TX") {
-		t.Fatalf("Returned wrong error response when begin failed: %v", err)
-	}
+type latestRootTest struct {
+	req         trillian.GetLatestSignedLogRootRequest
```

Comment:
`req` never varies, so in theory you could just drop it.

However, it looks like a get-root(invalid-tree-id) test gets dropped in the reshuffle so maybe keep `req` and reinstate a test case for that.

Reply:
It looked like it wasn't testing anything useful but having read it again I think it's testing an admin store failure, which can probably still use the same req.

Reply:
I've checked this out and the mock admin storage is never called so the removed test was not adding value. The admin storage would be called from storage, but it's also a mock and we bypass all that stuff.